lib/helpers.js

```diff
@@ -103,7 +103,7 @@ const diffToString = val => {
 };
 
 export default {
-	test,
+	// test,
```

Comment:
no such module - not sure if this is needed...

Reply:
whoops 😅